src/StackExchange.Redis/RedisDatabase.cs

```diff
@@ -997,6 +997,18 @@ public Task<long> ListLengthAsync(RedisKey key, CommandFlags flags = CommandFlag
             return ExecuteAsync(msg, ResultProcessor.Int64);
         }
 
+        public RedisValue ListMove(RedisKey sourceKey, RedisKey destinationKey, ListSide whereFrom, ListSide whereTo, CommandFlags flags = CommandFlags.None)
+        {
+            var msg = Message.Create(Database, flags, RedisCommand.LMOVE, sourceKey, destinationKey, whereFrom.ToString(), whereTo.ToString());
```

Comment:
`.ToString()` on these allocates on each call - instead we should use a ternary here which never allocates.